org.opent2t.sample.sensorpod.superpopular/org.opent2t.sample.sensorpod.superpopular.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.sensorpod.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T SuperPopular Sensor Pod",
```

Comment:
Sounds good.

---
In reply to: [79096733](https://github.com/openT2T/translators/pull/97#discussion_r79096733) [](ancestors = 79096733)